proto/firecracker.proto

```diff
@@ -81,5 +81,12 @@ message GetVMMetadataResponse {
 
 message JailerConfig {
     string NetNS = 1;
+
+	// The list of CPUs to run the jailed application on. This field can only
+	// be used on numa architectures
+    string CPUs = 2;
```

Comment:
Why aren’t we using “repeated string CPUs” if it is a list? Also, is there a more descriptive name than CPUs? I’m not familiar with the underlying numa api but does it call them anything more specific, even just like CPUID or something?

Same basic comment applies to “Mems” below

Reply:
This is a format that most people will be familiar with when using numa nodes which is a string to represent the CPU set and memory node sets. That's why I added the builder to help people that aren't familiar with the format.

I will update the docs to point to the format and add a little more context.

Reply:
Thanks! Makes much more sense now